pool/_RPC_dry.gno

```diff
@@ -4,42 +4,59 @@ import (
 	"std"
 
 	"gno.land/p/demo/common"
-	"gno.land/p/demo/u256"
+	"gno.land/p/demo/ufmt"
 
 	"gno.land/r/demo/consts"
+
+	i256 "gno.land/p/big/int256"
+	u256 "gno.land/p/big/uint256"
 )
 
 func DrySwap(
 	token0Path string,
 	token1Path string,
-	pFee uint16,
+	fee uint32,
 	recipient std.Address,
 	zeroForOne bool,
-	amountSpecified_ bigint,
-	sqrtPriceLimitX96_ bigint,
-) (bigint, bigint, bool) {
+	_amountSpecified string, // int256
+	_sqrtPriceLimitX96 string, // uint160
+) (string, string, bool) { // int256 x2
```

Comment:
I think this a generally a good practice to only use primitive types for public APIs, even if gno vm has no problem with dealing with pointers. :+1: